pywt/multidim.py

```diff
@@ -192,7 +192,7 @@ def idwt2(coeffs, wavelet, mode='sym'):
     for rowL, rowH in zip(L, H):
         data.append(idwt(rowL, rowH, wavelet, mode, 1))
 
-    return np.array(data, np.float64)
```

Comment:
I can open a separate PR that adds some tests to verify that the dtypes returned by various transforms are as expected.